ubcpi/ubcpi.py

```diff
@@ -510,9 +510,18 @@ def record_response(self, answer, rationale, status):
         answers = self.get_answers_for_student()
         stats = self.get_current_stats()
         truncated_rationle, was_truncated = truncate_rationale(rationale)
+        corr_ans_text = ''
+        if self.correct_answer == 3:    # handle scenario with no correct answer
```

Comment:
How do you know it is "3"? I thought the index will change when you add more answers.